app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module SessionsHelper
```

Comment:
Das hast du von meiner Seite? :)

Ich würde sowas heute im `ApplicationController` als private Methode und dann als `helper_method` einbauen. Jedenfalls für alle Helper, die im Controller und Views benutzt werden. Weil alles unter `app/helpers` nur in Views benutzt werden sollte.

Reply:
Ja, keine Ahnung Sessions Helper ist vor 1 Monat hinzugefügt worden :). Okay also für Helper alles unter ApplicationController.

Reply:
Ne, nicht alles, auf keinen Fall!

Es gibt klare Trennung: 
Helper sind nur für Views!
ApplicationController-Helper für Controller und Views.

Reply:
Wo ist da eine Trennung, wenn im ApplicationController-Helper Controller und Views drinnen sind und dem Helper nur Views ? Wozu nutzt ich, dann die Helper, wenn ich im ApplicationController beides Implementieren darf ?

Reply:
Das haben wir in der Vorlesung angeschaut. Du machst die Helper in Controller rein, wenn du die hauptsächlich in den Controller-Klassen nutzen willst. Eigentlich nicht in den Views. Es kommt aber vor, dass du die dann in beiden brauchst. Dann sind die Helper im ApplicationController besser aufgehoben. Wie alles, was du zwischen den Controllern teilen willst (Vererbung beachten).

Das ist aber selten der Fall! Daher, die meisten Helper-Methoden gehen auch in die Helper-Module.

Reply:
Jetzt verstehe ich was du meinst.